layouts/shortcodes/kubeflow-current-version.html

```diff
@@ -0,0 +1,1136 @@
+
```

Comment:
Oops I think something went wrong - maybe revert this change?

Reply:
Oops indeed! I've corrected it to just hold the version number as intended.